actionpack/lib/action_dispatch/system_testing/test_helpers/screenshot_helper.rb

```diff
@@ -39,13 +38,27 @@ def save_image
           end
 
           def display_image
-            if ENV["CAPYBARA_INLINE_SCREENSHOT"] == "artifact"
-              "\e]1338;url=artifact://#{image_path}\a"
-            else
+            message = "[Screenshot]: #{image_path}\n"
+
+            # Environment variables have priority
+            output_type = ENV["RAILS_SYSTEM_TESTING_SCREENSHOTS_OUTPUT"] || ENV["CAPYBARA_INLINE_SCREENSHOT"]
+
+            # If running in a CI environment, default to simple
+            output_type ||= "simple" if ENV["CI"]
+
+            # Default
+            output_type ||= "inline"
```

Comment:
Can you move output type to a method? I also think `RAILS_SYSTEM_TESTING_SCREENSHOTS_OUTPUT` is a really long env var so maybe something like `RAILS_SYSTEM_TESTING_SCREENSHOT`